core/src/test/java/com/klaytn/caver/legacy/feature/CustomGasTest.java

```diff
@@ -35,11 +36,11 @@
 import java.math.BigInteger;
 
 public class CustomGasTest {
-    private static final BigInteger GAS_PRICE = BigInteger.valueOf(10L);
+    private static BigInteger gasPrice;
```

Comment:
Maybe this also not used?

Reply:
Thanks. I checked other variables which is not used and remove them.